creator/extract_configs/templates/participant_conditions_config.py

```diff
@@ -0,0 +1,82 @@
+"""
+This is an extract configuration for a Participant Conditions file.
+
+See: https://www.notion.so/d3b/21e80815ba654a5fbc25f5a17b9e67a4?v=e8e02f33f87f47ff912350224cd5ab0f
+
+Required Columns:
+    Participant ID
+    Condition Name
+    Verification Status
+
+Optional Columns:
+    Age at Onset Value
+    Age at Onset Units
+    Age at Abatement Value
+    Age at Abatement Units
+    Condition Code
+    Body Site Name
+    Body Site Code
+"""
+
+from creator.extract_configs import common
```

Comment:
The Study Creator isn't meant to be an installable library and shouldn't be required to run ingest. This config won't work unless the user has study creator in their python path.

I think you want to inline all of the common functions within this template so that user's may use it as an extract config without any requirements other than the ingest library.

Reply:
What do you mean by users? Do we still expect Data Tracker users to be able to download these extract configs and run them in their ingest packages? 

I guess I thought we would only need ingest packages for studies that don't use these new templates. I would think that if the source data files conform to the templates then study creator would run the ingest, in which case, importing the ontology extract functions from outside the config, is ok. If we still expect users to be able to download and use these extract configs in their ingest packages then I understand why we want the common functions inline.

Reply:
The ontology extraction functions have been moved to the ingest library so this is no longer an issue